internal/services/compute/shared_image_gallery_resource.go

```diff
@@ -53,6 +55,64 @@ func resourceSharedImageGallery() *pluginsdk.Resource {
 				Optional: true,
 			},
 
+			"sharing_profile": {
```

Comment:
```suggestion
			"sharing": {
```
Per the [reference naming](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-naming.md) doc should we shorten this to just `sharing`?